hack/release/cmd/prerelease/prerelease.go

```diff
@@ -0,0 +1,101 @@
+package prerelease
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/drone/envsubst"
+	"github.com/spf13/cobra"
+)
+
+var Cmd *cobra.Command //nolint:gochecknoglobals // Cobra commands are global.
+
+const (
+	chartVersionFlagName          = "chart-version"
+	kommanderApplicationsFlagName = "kommander-applications-repo"
+	kommanderChartVersionTemplate = "${kommanderChartVersion:=%s}"
+
+	kommanderHelmReleasePathPattern        = "./services/kommander/*/kommander.yaml"
+	kommanderAppMgmtHelmReleasePathPattern = "./services/kommander-appmanagement/*/kommander-appmanagement.yaml"
+)
+
+func init() { //nolint:gochecknoinits // Initializing cobra application.
+	Cmd = &cobra.Command{
+		Use:   "pre-release",
+		Short: "Handles pre-release tasks for kommander-applications (i.e. updating Kommander chart versions)",
+		Args:  cobra.MaximumNArgs(0),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			chartVersion := Cmd.Flag(chartVersionFlagName).Value.String()
+			kommanderApplicationsRepo := Cmd.Flag(kommanderApplicationsFlagName).Value.String()
+			if _, err := os.Stat(kommanderApplicationsRepo); os.IsNotExist(err) {
+				return err
+			}
+			fullChartVersion := fmt.Sprintf(kommanderChartVersionTemplate, chartVersion)
+			err := updateChartVersions(kommanderApplicationsRepo, fullChartVersion)
+			if err != nil {
+				return err
+			}
+			fmt.Fprintf(cmd.OutOrStdout(), "Updated Kommander chart version to %s", chartVersion)
+			return nil
+		},
+	}
+	Cmd.Flags().String(chartVersionFlagName, "", "the new Kommander chart version")
+	Cmd.Flags().String(kommanderApplicationsFlagName, "", "the path to the local kommander-applications repository to modify")
+
+	err := Cmd.MarkFlagRequired(chartVersionFlagName)
```

Comment:
nit: I see `_ = cmd.MarkFlagRequired()` is used a lot in kommander-cli so guess we OK to do the same here to save 2 ifs?

Reply:
ah yeah, looking at the `MarkFlagRequired` code it looks like the only error that would returned is if the flag passed to it doesn't exist, which is unlikely since its being defined above 🤔. I'm cool with not bothering checking for that error if that seems reasonable